extensions/amp-auto-ads/amp-auto-ads.md

```diff
@@ -148,6 +148,11 @@ The fields to specify in the configuration object:
       </ul>
     </td>
   </tr>
+  <tr>
+    <td><code>adConstraints</code></td>
+    <td>AdConstraintsObj</td>
+    <td>An <em>optional</em> field that specifies the contraints that should be used when placing ads on the page.</td>
```

Comment:
Can we document that if `AdConstraintsObj` is not specified, the default one will be applied. And link to the default config?